src/app/core/sidenav/sidenav.component.ts

```diff
@@ -56,7 +56,7 @@ export class SidenavComponent implements OnInit, OnDestroy {
 
     this.appStateService.onRecordSelected().subscribe(resp => {
       this.rowId = resp.cinchyId;
-      this.filteredTableUrl = !this.rowId.toString().includes('null') ? `${this.tableUrl}?viewId=0&fil[Cinchy%20Id].Op=Equals&fil[Cinchy%20Id].Val=${this.rowId}` : this.tableUrl;
```

Comment:
Our forms ids can be a string, it's driven off the [Form ID] column. Row Id can also be the literal string "null".

Eg: <forms-url>?formId=myform&rowId=null

Will this be an issue?

Reply:
`?formId=myform&rowId=nullzzzzzzz` (anything other than `null` that isn't a valid/existing id) throws entitlements errors. Omitting `&rowId` altogether doesn't throw errors but will display the last loaded record - e.g. if you previously loaded `&rowId=3` - which is odd.

If you consider this to be in scope of these changes, ideally we'd show no form and a message communicating no record with that ID exists but that needs a bit more consideration so silently failing (don't show errors) and just showing the blank/new form would be sufficient.

Reply:
ignore the form id one, I misread the change. I guess you made it always string instead, that makes sense to me. But for the 'null' checks on rowId, would it cause an issue?

Reply:
"null" is no longer a possible value for the rowId. I've added logic to explicitly eliminate that case. I'll add in some more graceful handling for mangled rowIds.

Reply:
I'll also add in some better handling for cases where the session information conflicts with the current form for missing query params. In particular the case where the rowId isn't included, and the value stored in the session isn't valid for the current form.